ui-meta/workstation.yml

```diff
@@ -44,6 +44,9 @@ usage_template: |-
   The data volume is mounted at `/data`, and content stored there will persist for the lifetime
   of the workstation.
 
+  Note that this user storage (/home/azimuth) is EPHEMERAL and is removed on platform deletion, but is persistent
+  through platform patches. All system level changes outside of the home directory will be lost on a platform patch.
+
```

Comment:
I still don't think this scans quite right in the context of all of L44-L55. Maybe replace the whole of L44-L55 with something like:

```
The home directory for the platform user - `/home/azimuth` - persists for the lifetime of the
platform. **Content stored in this directory will be lost when the platform is deleted, but will
be retained during a patch operation**. All system-level changes outside of this directory will
be lost on a patch, including any packages installed using `apt`.

If configured by your Azimuth administrator, a project filesystem may be mounted at `/project`.
This filesystem is shared between all platforms deployed in the tenancy with `/project` directory
support - see the individual platform documentation. **Content stored in this directory will be
available to all such platforms, and is retained even when a platform is deleted.**
```

For example, now we are bind-mounting `/home/azimuth` into `/data` we no longer need to encourage users to use `/data` directly if they want their data to persist.

Reply:
Yes I think that reads better, though do we need to mention the existence of the /data volume still?